client/components/jetpack/licensing-activation-banner/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import { getSiteAdminUrl } from 'calypso/state/sites/selectors';
+import { getUserLicensesCounts } from 'calypso/state/user-licensing/selectors';
+import keyIcon from './key-icon.svg';
+
+import './style.scss';
+
+interface Props {
+	siteId: number;
+}
+
+function LicensingActivationBanner( { siteId }: Props ) {
+	const translate = useTranslate();
+	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
+	const jetpackDashboardUrl = siteAdminUrl + 'admin.php?page=jetpack#/my-plan';
+	const userLicensesCounts = useSelector( getUserLicensesCounts );
+	const hasDetachedLicenses = userLicensesCounts && userLicensesCounts[ 'detached' ] !== 0;
+
+	if ( hasDetachedLicenses ) {
+		return (
+			<>
+				<div className="licensing-activation-banner">
+					<img className="licensing-activation-banner__key-icon" src={ keyIcon } alt="" />
+					{ translate( 'You have an available product license key. ' ) }
```

Comment:
Is it intentional to have that extra space at the end of this string? It looks like the "Activate it now" text appearance isn't depending on it.

Reply:
This was not intentional and has been updated.